examples/tutorials/advance_model_config/01_microbatch.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<p>Mention this tradeof in the very beginning of the tutorial</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>